buildSrc/src/main/kotlin/DocsLandingPage.kt

```diff
@@ -0,0 +1,53 @@
+import org.gradle.api.Project
+import software.amazon.smithy.utils.CodeWriter
+import java.io.File
+
+/**
+ * Generate a basic docs landing page into [outputDir]
+ *
+ * The generated docs will include links to crates.io, docs.rs and GitHub examples for all generated services. The generated docs will
+ * be written to `docs.md` in the provided [outputDir].
+ */
+fun Project.docsLandingPage(awsServices: List<AwsService>, outputDir: File) {
+    val project = this
+    val writer = CodeWriter()
+    with(writer) {
+        write("# AWS SDK for Rust")
+        write(
+            """The AWS SDK for Rust contains one crate for each AWS service, as well as ${docsRs("aws-config")},
+            |a crate implementing configuration loading such as credential providers.""".trimMargin()
+        )
+
+        writer.write("## AWS Services")
+        /* generate a basic markdown table */
+        writer.write("| Service | [docs.rs](https://docs.rs) | [crates.io](https://crates.io) | [Usage Examples](https://github.com/awslabs/aws-sdk-rust/tree/main/examples/) |")
+        writer.write("| ------- | ------- | --------- | ------ |")
+        awsServices.forEach {
+            writer.write(
+                "| ${it.humanName} | ${docsRs(it)} | ${cratesIo(it)} | ${
```

Comment:
Do we want to order this by human name or by crate name? If the latter, then I think maybe we should put the crate name on the left rather than the human name.